src/renderer/entities/proxy/ui/ProxyAccount/ProxyAccount.tsx

```diff
@@ -1,11 +1,20 @@
 import { cnTw, toAddress, toShortAddress } from '@shared/lib/utils';
 import { BodyText, DropdownIconButton, HelpText, Identicon, Truncate } from '@shared/ui';
-import { AccountId } from '@shared/core';
-import { ProxyType } from '../../lib/types';
-import { ProxyTypeName } from '@entities/proxy/lib/constants';
+import { AccountId, ProxyType } from '@shared/core';
 import { useI18n } from '@app/providers';
 import { DropdownIconButtonOption } from '@shared/ui/types';
 
+const ProxyTypeName: Record<ProxyType, string> = {
```

Comment:
I think we might need to use it in other places so for now I think it's better to leave it constants file with export